New version: OceanAnalysis v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: OceanAnalysis
- Repository: https://github.com/dankelley/OceanAnalysis.jl
- Created by: @dankelley
- Version: v0.1.0
- Commit: 6ffb055cf95fb7a2f7289e084bce71fbbba239d3
- Reviewed by: @dankelley
- Reference: https://github.com/dankelley/OceanAnalysis.jl/commit/6ffb055cf95fb7a2f7289e084bce71fbbba239d3#commitcomment-167205294
- Description: Julia code for oceanographic analysis
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
### Changed
- improve consistency of `;` placement in functions
- improve code documentation, e.g. fixing erroneous line breaks in list items
- BREAKING: rename `getElement()` as `get_element()`.
- BREAKING: rename `plotProfile()` as `plot_profile()`.
- BREAKING: rename `plotTS()` as `plot_TS()`.
- BREAKING: rename `readArgo()` as `read_argo()`.
- BREAKING: rename `readCtdCnv()` as `read_ctd_cnv()`.
### Added
- add `coastline()`, `plot_coastline()` and `plot_coastline!()`
- add `get_amsr_file()`, `read_amsr()` and `plot_amsr()`
- add `get_topography_file()`, `read_topography()` and `plot_topography()`
- add `N2()`
- add `get_argo_index()`, `get_argo_file()` and `read_argo()`
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd61b2c7295bd38ad6544ee74256c4aee5b0a8f070a697e279349d1267624f6205be3db169d859036b341878f6efad641f9a4b1ce5f9a2e1fc594eb014777f58bc55051035b617d2e18aecbadc7fcf83e1d8ef4ca097b3fdc74588b7d88641a9879d0b9c60de74b5954151d649e5390faf1e6238a02eede9b323f1467adb37b70ae318bdd99eb6883048cd7bc1be091c751d6e46adef886c497fcacfc8ea1d1079e20c09a352691e9c5d562c1733667c537 -->

dankelley commented:
@JuliaRegistrator register
Release Notes
- Add plot_topography().
- Work towards more helpful ';' usage in functions.
- Clean up code documentation, e.g. fixing line breaks in sentences.
- Add a 'dep' item for Plots.

There are no breaking changes.

JuliaRegistrator committed:
New version: OceanAnalysis v0.1.0

UUID: 935d77d6-d85b-49cc-a465-c336e8287d34
Repo: https://github.com/dankelley/OceanAnalysis.jl.git
Tree: b1af4b9a06bf8c6a9d06261aceda410db87824cc

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
